client/src/modules/Registry/components/AdditionalInfo/index.test.tsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable testing-library/no-unnecessary-act */
```

Comment:
Are you sure about disabling this rule for entire file?

Reply:
Have you checked the article that I sent previously? https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily

> what they don't know is that render and fireEvent are already wrapped in act

Reply:
I did, it ain't work without act, it has something to do with antd form state under the hood. Any way, I switched to user-event here, and it works fine without `act`